project.clj

```diff
@@ -24,7 +24,7 @@
 
   :test-paths ["test/common"] ;; See `test-clj` and `test-cljs` profiles below.
 
-  :profiles {:provided {:dependencies [[org.clojure/clojure "1.7.0"]]}
+  :profiles {:provided {:dependencies [[org.clojure/clojure "1.9.0-alpha10"]]}
```

Comment:
Well, we do run the tests on all supported clojure versions, not just the provided one. And you can always override the provided `clojure` when running `cider-nrepl` locally.